docker/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM python:3.7
+
+# Use UTF-8 as the default locale
+ENV LANG=C.UTF-8
+
+# Better movement with [Ctrl + →] and [Ctrl + ←]
+ENV INPUTRC=/etc/inputrc
+
+RUN set -ex \
+  && echo '"\e[1;5C": forward-word' >> /etc/inputrc \
+  && echo '"\e[1;5D": backward-word' >> /etc/inputrc
+
+# Set working directory.
+# Mount your project under the same path.
+WORKDIR /usr/src
+
+# Install useful programs
+RUN apt-get update -y && apt-get install -y \
+      zsh \
+      less \
+      git
+
+# Set Zsh as the default shell
+SHELL ["/usr/bin/zsh", "-c"]
+
+# Install DVC with your specified files
+RUN pip install "dvc[all]"
+
+# Enable autocompletion scripts for DVC
+RUN wget \
+      -O /usr/local/share/zsh/site-functions/_dvc \
+      https://raw.githubusercontent.com/iterative/dvc/master/scripts/completion/dvc.zsh
+
+# Add a user to run DVC
+ARG USER_ID=1000
+RUN useradd --create-home -u ${USER_ID} dvc
+USER dvc
+
+# Load and rice the completion menu with `zstyle`:
+# http://zsh.sourceforge.net/Doc/Release/Zsh-Modules.html#The-zsh_002fzutil-Module
+RUN echo "autoload -U compinit && compinit" >> $HOME/.zshrc \
+      && echo "zstyle ':completion:*:*:*:*:*' menu select" >> $HOME/.zshrc \
+      && echo "zstyle ':completion:*:matches' group 'yes'" >> $HOME/.zshrc \
+      && echo "zstyle ':completion:*:options' description 'yes'" >> $HOME/.zshrc \
+      && echo "zstyle ':completion:*:options' auto-description '%d'" >> $HOME/.zshrc \
+      && echo "zstyle ':completion:*:corrections' format ' %F{green}-- %d (errors: %e) --%f'" >> $HOME/.zshrc \
+      && echo "zstyle ':completion:*:descriptions' format ' %F{yellow}-- %d --%f'" >> $HOME/.zshrc \
+      && echo "zstyle ':completion:*:messages' format ' %F{purple} -- %d --%f'" >> $HOME/.zshrc \
+      && echo "zstyle ':completion:*:warnings' format ' %F{red}-- no matches found --%f'" >> $HOME/.zshrc \
+      && echo "zstyle ':completion:*:default' list-prompt '%S%M matches%s'" >> $HOME/.zshrc \
+      && echo "zstyle ':completion:*' format ' %F{yellow}-- %d --%f'" >> $HOME/.zshrc \
+      && echo "zstyle ':completion:*' group-name ''" >> $HOME/.zshrc \
+      && echo "zstyle ':completion:*' verbose yes" >> $HOME/.zshrc
```

Comment:
really if all of this is genuinely useful for `zsh` users then surely it should be under the `scripts/` directory just like `completion/`?

Reply:
@casperdcl , it is usually provided by Zsh _prompts_ like "pure" or _frameworks_ like "oh-my-zsh".

so you don't need to deal with `zstyle` and stuff like that :)

Reply:
In that case not really sure it belongs here. At the most maybe use pure or omz?

Basically I feel either put it in the scripts folder or remove it completely from the dockerfile.

Reply:
Right, let's remove this, then; just for context, this is how it looks like

with `zstyle`:
![2019-11-25-20:08:35](https://user-images.githubusercontent.com/7363250/69593566-29a3eb00-0fbf-11ea-8391-caa147e610ff.png)


without `zstyle` (no way to iterate between the options):
![2019-11-25-20:10:26](https://user-images.githubusercontent.com/7363250/69593671-869fa100-0fbf-11ea-9c86-36d6f968f582.png)

Reply:
![2019-11-25-20:18:36](https://user-images.githubusercontent.com/7363250/69594016-9d92c300-0fc0-11ea-8ed9-9b5f4e1254fa.png)
boring bash :stuck_out_tongue: